### ISSUE TITLE
[Bug] Komga merge source doesn't seem to merge consistently

### ISSUE THREAD

**Issue by farahnur42** [CarlosEsco/Neko#1244]
### Steps to reproduce

My test scenario is as follows:
1. Log into my Komga server (no reverse proxy, no SSL setup and no public DNS entry either)
2. Try to merge a Komga title
3. Nothing happens (I've managed to get it working once and only once in 6 or so tries)

### Expected behavior

It should merge the books from the Komga entry

### Actual behavior

It does nothing

### Neko version

2.11.7

### Android version

Android 12

### Device

Redmi Note 10 Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.
